lib/elixir/lib/list.ex

```diff
@@ -872,9 +872,9 @@ defmodule List do
 
         To be converted to a string, a list must contain only:
 
-          * strings
-          * integers representing Unicode code points
-          * or a list containing one of these three elements
+          * strings;
+          * integers representing Unicode code points;
+          * and/or a list containing one of these three elements.
```

Comment:
Could we achieve the same result by removing "or" from what we had here before the change?

Reply:
Nope. because that would be that it must contain all three items

Reply:
We could remove "or" and in the header say "a list must contain either:"

Reply:
I'm not really sure how so, but then what if we make the first sentence "To be converted to a string, a list must only contain one of the following:"?

Reply:
I like it. let's go with it.

Reply:
fixed,

Reply:
actually it is wrong. "one of the following" is wrong. it can contain a string and a number

Reply:
I'm changing it for "To be converted to a string, a list must either be empty or only contain the following elements:"